src/find-build.js

```diff
@@ -15,7 +16,8 @@ const findBuild = async () => {
   const buildJob = workflowJobs.find(j => j.name === 'build' && j.status === 'success')
 
   const jobArtifacts = await fetch(`${base}/project/${projectSlug}/${buildJob.job_number}/artifacts`).then(r => r.json()).then(o => o.items)
-  const artifactUrl = jobArtifacts[0].url
+  const buildArtifact = jobArtifacts.find(artifact => artifact.path === buildArtifactPath)
+  const artifactUrl = buildArtifact.url
```

Comment:
[Nit] Could be written as

```suggestion
  const { url: buildArtifact } = _.find({ path: buildArtifactPath }, jobArtifacts)
```

for concision.

Reply:
I went with the function instead of object for consistency with other uses of `find` in this file.